pkg/restapi/verifier/operation/operations_test.go

```diff
@@ -20,6 +20,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/trustbloc/edge-service/pkg/restapi/model"
+
```

Comment:
extra blank line, also group edge-service imports with the rest of edge-service imports block below

Reply:
Done